recipes/AGDC/recipe.py

```diff
@@ -0,0 +1,41 @@
+"""Author: Norland Raphael Hagen - 09-16-2021
+Pangeo-Forge recipe for Australian Gridded Climate Data (AGCD) V2 precipitation, temperature and vapour pressure data"""  # noqa: E501
+
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern, MergeDim
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+
+# Filename Pattern Inputs
+target_chunks = {"lat": 3451, "lon": 4426, "time": 20}
```

Comment:
These chunks seems pretty big? Could we get away with smaller ones? That might help the locking situation.